app/views/shared/_alerts.html.erb

```diff
@@ -2,7 +2,8 @@
   <% if message = flash[type] %>
     <div class="alert <%= bootstrap_alert_class_for(type) %> alert-dismissible" role="alert">
```

Comment:
I copied this entire file (and the helper method in application_helper directly from summary generator. Happy to change. :)